lms/templates/learner_dashboard/_dashboard_incomplete_profile_notification.html

```diff
@@ -0,0 +1,20 @@
+<%page expression_filter="h"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import Text
+%>
+<div class="profile-sidebar" role="region" aria-labelledby="account-activation-title">
+  <div class="user-info">
+    <ul>
+      <li class="status status-verification warning" aria-labelledby="status-title" tabindex="-1">
+        <span id="status-title" class="title status-title">
+          <i class="fa fa-exclamation-circle" aria-hidden="true"></i>
+          <span class="sr">Notice</span>
+          ${_("Your profile has missing fields!")}
+        </span>
+        <p class="status-note">${Text(_("Please, complete your profile."))}
```

Comment:
Could you please link to the user's "Account" page, so they can fill in their profile easily?  e.g. https://campus-dev.opencraft.hosting/account/settings